site/content/en/docs/Reference/api.md

```diff
@@ -582,6 +676,44 @@ Open Match does not require or inspect any fields on assignment.
 
 
 
+<a name="openmatch.Backfill"></a>
+
+### Backfill
+Represents a backfill entity which is used to fill partially full matches.
+
+BETA FEATURE WARNING:  This call and the associated Request and Response
+messages are not finalized and still subject to possible change or removal.
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| id | [string](#string) |  | Id represents an auto-generated Id issued by Open Match. |
+| search_fields | [SearchFields](#openmatch.SearchFields) |  | Search fields are the fields which Open Match is aware of, and can be used when specifying filters. |
+| extensions | [Backfill.ExtensionsEntry](#openmatch.Backfill.ExtensionsEntry) | repeated | Customized information not inspected by Open Match, to be used by the match making function, evaluator, and components making calls to Open Match. Optional, depending on the requirements of the connected systems. |
```

Comment:
matchmaking function ?

Reply:
with this you mean to delete the space between match and making, or you mean mmf should not go here??

Reply:
delete the space